recipes/holoscan/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+set -x
+
+check-glibc bin/* lib/* lib/ucx/* lib/gxf_extensions/*
+find python/ -name "*.so*" | xargs -I"{}" check-glibc "{}"
+
+cp -rv bin $PREFIX/
```

Comment:
Why are we unpacking a redist instead of building from source when the source is available?

https://github.com/nvidia-holoscan/holoscan-sdk

Reply:
I see that we already discussed this above back in December 2024.

> For gds-tools, onnxruntime and torchvision I will see when we can get respective owners to provide missing DSO, and for TensorRT we'll have to find out a plan for it with respect to conda.

Have you opened issues on the feedstocks for these packages asking for the relevant libraries?